inc/compatibility/base.php

```diff
@@ -195,15 +194,15 @@ protected function run_fix_everytime() {
 	 * @return void
 	 */
 	protected function persist_data( array $plugin_methods = [] ) {
-		$pantheon_applied_fixes = get_option( 'pantheon_applied_fixes' ) ?: [];
-		$old = $pantheon_applied_fixes[ static::$plugin_slug ] ?? [];
+		$pantheon_applied_fixes                         = get_option( 'pantheon_applied_fixes' ) ?: [];
+		$old                                            = $pantheon_applied_fixes[ static::$plugin_slug ] ?? [];
```

Comment:
nit: the old spacing was fine, no need to align these. it shouldn't break linting if you're using Pantheon WP Coding Standards.
```suggestion
		$pantheon_applied_fixes = get_option( 'pantheon_applied_fixes' ) ?: [];
		$old = $pantheon_applied_fixes[ static::$plugin_slug ] ?? [];
```